clang/lib/Tooling/DependencyScanning/DependencyScanningFilesystem.cpp

```diff
@@ -362,7 +357,7 @@ DependencyScanningWorkerFilesystem::openFileForRead(const Twine &Path) {
   SmallString<256> OwnedFilename;
   StringRef Filename = Path.toStringRef(OwnedFilename);
 
-  if (Filename.ends_with(".pcm"))
+  if (shouldBypass(Filename))
```

Comment:
Are the .pcm files to be generated in the modules cache path?

Reply:
All implicitly-built PCM files, yes. We'll now start caching explicitly-built PCM files (that were either passed on the command line or extracted from a PCH file), but these are immutable anyways, so that should be fine.